baltic-honeybadger/2018/_index.es.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Austin Bitcoin Developers
+---
+
+{{< childpages >}}
```

Comment:
I believe you need two index files if you are going to next 2018 instead baltic-honeybadger.